roles/get-sitename/tasks/main.yml

```diff
@@ -155,7 +155,7 @@
   set_fact:
     possible_site_ids: "{{ sites_formatted|json_query(existing_sites_query) }}"
   vars:
-    existing_sites_query: "[?site=='{{ acsf_site_name }}'].id"
+    existing_sites_query: "[?site=='{{ acsf_site_name|lower }}'].id"
```

Comment:
Not sure this is needed but everything is lowercase in sites.json so this can't hurt.